unit-tests/functional/utilities.h

```diff
@@ -0,0 +1,97 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+#include "../tests.h"
+#include "../include/librealsense2/rs.hpp"
+#include "../include/librealsense2/hpp/rs_context.hpp"
+#include "../include/librealsense2/hpp/rs_internal.hpp"
+#include "concurrency.h"
+
+using namespace rs2;
+
+std::vector< stream_profile > find_profiles_to_stream( rs2::depth_sensor depth_sens )
```

Comment:
Needs docs.
It just finds the first profiles? I can see how we'd want to have it accept a default resolution and FPS, maybe they can be parameters later.
Why is confidence required?
Also, it just finds the depth profiles. I think the name should reflect that. You imagine there will be a parallel function that does this for RGB?

Reply:
Split to 3 functions find_default_depth_profile, find_default_ir_profile and find_confidence_corresponding_to_depth because confidence is not defined as default profile.